src/views/Submit.vue

```diff
@@ -236,6 +238,26 @@ export default {
 			this.fetchFullForm(this.form.id)
 		}
 		SetWindowTitle(this.formTitle)
+		if (localStorage.getItem(`${this.shareHash}`)) {
+			this.formValuesForLocalStorage = JSON.parse(localStorage.getItem(`${this.shareHash}`))
```

Comment:
```suggestion
		if (localStorage.getItem(`${this.hash}`)) {
			this.formValuesForLocalStorage = JSON.parse(localStorage.getItem(`${this.hash}`))
```
Should be just "hash". Otherwise the values are stored with an empyt key ""

Reply:
One more thing: I think that we should also add some prefix to the storage key like `nextcloud_forms_`